packages/appstore/src/components/wallets-banner/wallets-banner.scss

```diff
@@ -0,0 +1,57 @@
+.wallets-banner {
+    &__container {
+        position: relative;
+        padding-inline: 4.8rem;
+        margin-bottom: 2.4rem;
+
+        border: 1px solid #e6e9e9;
+        box-shadow: 0px 12px 16px -4px rgba(14, 14, 14, 0.08), 0px 4px 6px -2px rgba(14, 14, 14, 0.03);
```

Comment:
I think we have a variable for box-shadow. Use variable instead of hard-code value.